collection/collection/src/commonTest/kotlin/androidx/collection/ArraySetTest.kt

```diff
@@ -15,98 +15,15 @@
  */
 package androidx.collection
 
-import androidx.collection.internal.Lock
-import androidx.collection.internal.synchronized
-import kotlin.random.Random
 import kotlin.test.Test
 import kotlin.test.assertEquals
 import kotlin.test.assertFalse
 import kotlin.test.assertNotEquals
 import kotlin.test.assertTrue
-import kotlinx.coroutines.CoroutineScope
-import kotlinx.coroutines.DelicateCoroutinesApi
-import kotlinx.coroutines.Dispatchers
-import kotlinx.coroutines.ExperimentalCoroutinesApi
-import kotlinx.coroutines.cancel
-import kotlinx.coroutines.delay
-import kotlinx.coroutines.isActive
-import kotlinx.coroutines.launch
-import kotlinx.coroutines.newFixedThreadPoolContext
-import kotlinx.coroutines.runBlocking
 
 internal class ArraySetTest {
     private val set = ArraySet<String>()
 
-    /**
-     * Attempt to generate a ConcurrentModificationException in ArraySet.
-     *
-     *
-     * ArraySet is explicitly documented to be non-thread-safe, yet it's easy to accidentally screw
-     * this up; ArraySet should (in the spirit of the core Java collection types) make an effort to
-     * catch this and throw ConcurrentModificationException instead of crashing somewhere in its
-     * internals.
-     */
-    @Suppress("UnnecessaryOptInAnnotation")
-    @OptIn(
-        ExperimentalCoroutinesApi::class, // newFixedThreadPoolContext is experimental in common
-        DelicateCoroutinesApi::class, // newFixedThreadPoolContext is delicate in jvm
-    )
-    @Test
-    fun testConcurrentModificationException() {
```

Comment:
@MatkovIvan, a note for the future:

Besides enabling compilation of `commonMain` with stubs in AOSP, we also need to do the same for `commonTest` (as a separate step)

P.S. Maybe we even don't need stubs and publication. We need to test `metadataCommonMainClasses` (and similar command for commonTest) . Another target is still needed to be defined, but sources with actual`'s aren't needed.